src/storage/storageViewer.js

```diff
@@ -14,27 +21,21 @@ class StorageViewer {
    * @param {Function} - callback - contains a map: [hashedKey] = {key, hashedKey, value}
    */
   storageRange (callback) {
-    this.storageResolver.storageRange(this.context.tx, this.context.stepIndex, callback)
+    this.storageResolver.storageRange(this.context.tx, this.context.stepIndex, this.storageChanges, this.context.address, callback)
```

Comment:
Is it better to always hand over `this.storageChanges` or rather search inside `this.storageChanges` first and let the `storageResolver` only handle the "static" storage?